packages/mirrorful/editor/src/components/ColorPalette/utils.ts

```diff
@@ -71,7 +71,11 @@ export const handleInvalidColor = (input: string) => {
   }
 
   const validSubsetRegex = /^#?[0-9A-Fa-f]{0,6}$/ // regex to validate if input is a valid subset of a hexcode
-  const randomHex = Math.floor(Math.random() * 16777215).toString(16) // generate a random valid hexcode
+
+  let randomHex :String = ''
+  for(let i:number = 0; i < 6; i++){
```

Comment:
let's try and figure out why your `prettier` is still not working (be sure to install extension + turn on autoformat on save), but this is ✅